server/src/test/java/org/opensearch/index/IndexSettingsTests.java

```diff
@@ -901,7 +847,7 @@ public void testRemoteRepositoryExplicitSetting() {
             Settings.builder()
                 .put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT)
                 .put(IndexMetadata.SETTING_REMOTE_STORE_ENABLED, true)
-                .put(IndexMetadata.SETTING_REMOTE_STORE_REPOSITORY, "repo1")
+                .put(IndexMetadata.SETTING_REMOTE_SEGMENT_STORE_REPOSITORY, "repo1")
```

Comment:
Does it mean it is possible to enable remote store and pass only segment store repository?

Reply:
no, but i think these tests are just for verifying individual functionality. we would have a test to validation index creation failure in this case separately in metadata create index service or createRemoteIndexIt

Reply:
okay, as long as we are not allowing passing only remote segment repository and have test/s to cover this, it is fine.